datastream-bigquery/src/main/java/com/linkedin/datastream/bigquery/BatchBuilder.java

```diff
@@ -99,19 +72,14 @@ public void run() {
             try {
                 if (aPackage.isDataPackage()) {
                     _registry.computeIfAbsent(aPackage.getTopic() + "-" + aPackage.getPartition(),
```

Comment:
I see that the registry is keyed with topic+partition. 
It seems like your exception records and normal records related to a specific topic+partition end up in the same batch.

I don't see that they are directed to different tables in the committer. Do we have a collision here?

Reply:
For the exception records, we set a custom topic name based on the exceptions table associated with the topic, so the exception records will get routed to their own batches. Looking at the code now, I noticed that I am using the hard-coded exceptions table suffix instead of the new configurable exceptions table name. The hard-coded suffix isn't breaking anything, but it's probably best to use the configured exceptions table name.

Reply:
I see it now. Thanks for confirming.
exceptionRecordMetadata.put(KAFKA_ORIGIN_TOPIC, getExceptionsTopicName(topic));